generators/app/templates/client/public/index.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title><%= _.kebabCase(appName) %></title>
+  </head>
+  <body>
+    <div id="root"></div>
+    <script src="/bundle.js"></script>
```

Comment:
I had used the `assets` directory so that there was no chance of conflicts with the react-router routing, the api routing and the assets routing.

Don't you think that like this with everything mixed it gets kinda messy?